src/coreclr/gc/gc.cpp

```diff
@@ -28947,6 +28960,12 @@ void heap_segment::thread_free_obj (uint8_t* obj, size_t s)
         }
 
         free_list_tail = obj;
+
+        free_list_size += s;
+    }
+    else
```

Comment:
@PeterSolMS Is this a typo? I don't see an `if` above.

Reply:
it's 18 lines above  - `    if (s >= min_free_list)`

Reply:
Oh gosh. Bug in github. It doesn't show me anything above 28960 and skips directly to 28947 - I missed that.

Reply:
codeflow FTW :)